cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -594,6 +594,17 @@ def master_services_down():
             failing_services.append(service)
     return failing_services
 
+def add_systemd_file_limit():
+    directory = '/etc/systemd/system/snap.kube-apiserver.daemon.service.d'
+    file_name = 'file-limit.conf'
+    path = os.path.join(directory, file_name)
+    if not os.path.isfile(path):
+        with open(path, 'w') as f:
+            f.write('[Service]\n')
+            f.write('LimitNOFILE=65535')
+
+    check_call(['systemctl', 'daemon-reload'])
```

Comment:
we end up calling daemon-reload twice in a row. Once for add_systemd_restart_always and once for add_systemd_file_limit. Should we pull this up into start_master instead?

Reply:
I agree